scripts/publish-plugin.sh

```diff
@@ -16,28 +19,70 @@ if [ "$(go env GOOS)" = "windows" ]; then
     BIN_SUFFIX=".exe"
 fi
 
-(cd "${ROOT}/provider" && go build \
-   -ldflags "-X github.com/pulumi/pulumi-openstack/provider/v2/pkg/version.Version=${VERSION}" \
-   -o "${WORK_PATH}/pulumi-resource-openstack${BIN_SUFFIX}" \
-   "${ROOT}/cmd/pulumi-resource-openstack")
+(cd provider && go build \
```

Comment:
Every provider repo has a `publish-package.sh` file that has been copied from somewhere else. This repo was using an earlier version that didn't define `${PROVIDER_NAME}` and just hard-coded it where needed.